chef/cookbooks/nfs-server/recipes/default.rb

```diff
@@ -40,12 +40,14 @@
       command "chmod 755 /etc/init.d/nfs-kernel-server"
     end
   end
-
-when "centos","redhat","suse"
+when "rhel"
   package "nfs-utils"
-  if node[:platform_version].to_f >= 6 || node[:platform] == "suse"
-    rpcService="rpcbind"
+  if node[:platform_version].to_f >= 6
+    rpcService = "rpcbind"
   end
+when "suse"
+  package "nfs-utils"
+  rpcService = "rpcbind"
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)